src/states/peer.rs

```diff
@@ -180,7 +180,7 @@ impl Peer {
 
     #[cfg_attr(feature = "cargo-clippy", allow(too_many_arguments))]
     pub fn from_bootstrapping(
-        our_section: (Prefix, BTreeSet<PublicId>),
+        our_section: &(Prefix, BTreeSet<PublicId>),
```

Comment:
Might be better to change this to `our_section: (Prefix, &BTreeSet<PublicId>)`, so the called doesn't have to clone the `BTreeSet` when passing it in, as it doesn't seem to be consumed inside this function.